crates/core/src/config/mod.rs

```diff
@@ -164,36 +134,62 @@ impl Config {
     /// Returns a `Option<PathBuf>` to the WTF directory.
     /// This will return `None` if no `wow_directory` is set in the config.
     pub fn get_wtf_directory_for_flavor(&self, flavor: &Flavor) -> Option<PathBuf> {
-        let dir = self.wow.directories.get(flavor);
-        match dir {
-            Some(dir) => {
-                // The path to the WTF directory
-                let mut addon_dir = dir.join("WTF");
+        self.get_directory_for_flavor(flavor, "WTF")
+    }
+
+    /// Returns a `Option<PathBuf>` to the Screenshots directory.
+    /// This will return `None` if no `wow_directory` is set in the config.
+    pub fn get_screenshots_directory_for_flavor(&self, flavor: &Flavor) -> Option<PathBuf> {
+        self.get_directory_for_flavor(flavor, "Screenshots")
+    }
+
+    /// Return a `Option<PathBuf` to a directory.
+    /// This will return none if no `wow_directory` is set in the config.
+    fn get_directory_for_flavor(&self, flavor: &Flavor, relative_path: &str) -> Option<PathBuf> {
+        let wow_flavor_dir = self.wow.directories.get(flavor);
+        match wow_flavor_dir {
+            Some(wow_flavor_dir) => {
+                let mut dir = wow_flavor_dir.join(relative_path);
 
                 // If path doesn't exist, it could have been modified by the user.
```

Comment:
```suggestion
                // If dir doesn't exist, it could have been modified by the user.
```
Small nitpick now we use the word `dir `instead of `path`

Reply:
Fixed